Aquarius/Src/Aquarius/Core/Log.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "easylogging++.h"
+
+// Note: for printf-like formatting, use only %v instead of type specific identifiers within log string
+
+// Core log macros
+#define AQ_CORE_TRACE(String, ...)		::Aquarius::Log::getCoreLogger()->trace(String, __VA_ARGS__);
+#define AQ_CORE_INFO(String, ...)		::Aquarius::Log::getCoreLogger()->info(String, __VA_ARGS__);
+#define AQ_CORE_WARNING(String, ...)	::Aquarius::Log::getCoreLogger()->warn(String, __VA_ARGS__);
```

Comment:
Fix all ::Aquarius to be inline with each other.

Reply:
Check this in your text editor, I believe they are inline, github tends to be funny when tabs and spaces are mixed together

Reply:
Nvm fixed this by changing tabs to spaces, thanks visual studio for doing weird shit